portal-impl/src/content/Language_hu.properties

```diff
@@ -5751,7 +5751,7 @@ x-item-is-ready-to-be-moved-to-x={0} elem áthelyezhető ide: <em>{1}</em>.
 x-items-are-ready-to-be-moved={0} elemek áthelyezhetők.
 x-items-are-ready-to-be-moved-to-x={0} elem áthelyezhető ide: <em>{1}</em>.
 x-items-are-selected={0} elem kijelölve.
-x-items-per-page=Tételek oldalanként
+x-items-per-page={0} tétel oldalanként
```

Comment:
Thanks for fixing the Hungarian translations.
To avoid future misunderstandings it would be better to have one commit fixing the placeholder issue, and another commit to fix Hungarian translation.